examples/opf/clients/nyctaxi/nyctaxi_anomaly.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2016, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+"""
+A simple client to create a HTM anomaly detection model for nyctaxi dataset.
+The script prints out all records that have an abnormally high anomaly
+score.
+"""
+
+import csv
+import datetime
+import logging
+import numpy as np
+import pandas
+
+from pkg_resources import resource_filename
+
+from nupic.frameworks.opf.modelfactory import ModelFactory
+
+import model_params
+
+_LOGGER = logging.getLogger(__name__)
+
+_INPUT_DATA_FILE = resource_filename(
+  "nupic.datafiles", "extra/nyctaxi/nyc_taxi.csv"
+)
+_OUTPUT_PATH = "anomaly_scores.csv"
+
+_ANOMALY_THRESHOLD = 0.9
+
+
+def getCSVData(dataPath):
+  try:
+    data = pandas.read_csv(dataPath)
```

Comment:
We don't have a dependency on pandas in NuPIC. Just use the built in `csv` package and manually parse the values.